client/src/components/TabularViewComponent/TabularViewComponent.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+@import "../../theme/theme.scss";
+
+.ms-DetailsList {
+    overflow-x: hidden;
+}
+
+.ms-Viewport  {
+    width: 90%;
```

Comment:
Why 90% specifically?

Reply:
![image](https://user-images.githubusercontent.com/97867829/155812915-0dfc7856-b3f0-4589-99b4-1b67f7417fd8.png)
I was just ask to stay as close to the design as possible

Reply:
Exept the last column in the design is actual cut, so since for the time being there is no intention to put something on the right I'm just using a size that looks nice to the current data

Reply:
Can you share a screenshot of the implementation with the current data?

Reply:
![image](https://user-images.githubusercontent.com/97867829/155813630-36e7648a-603c-43d5-8576-e4ff3922696b.png)

Reply:
I'd be inclined not to constrain the table to 90%. I.e. just let it stretch across the available space.

Reply:
Done 
![image](https://user-images.githubusercontent.com/97867829/155813831-a70d3565-297e-456a-81d2-ce399680e87e.png)